config/locales/en/apply/student.yml

```diff
@@ -11,39 +11,39 @@ en:
         As a member, you will join a family of fellow students working to use our education
         and skills to make the world a better place.
       applications_closed: >
-        Our student applications for the Spring 2023 semester are now closed. Sign up for our mailing list or follow our <a href="https://www.facebook.com/CalBlueprint/" target="_blank" rel="noopener noreferrer">Facebook page</a> to stay up to date on Fall 2023 applications!
+        Our student applications for the Spring 2024 semester will open Tuesday, January 16th, and close on Thursday, January 25th at 11:59 PM PDT (midnight). Sign up for our mailing list or follow our <a href="https://www.facebook.com/CalBlueprint/" target="_blank" rel="noopener noreferrer">Facebook page</a> to stay up to date!
       developer_description: Learn what it means to be a developer <strong>👉 <a href="https://go.calblueprint.org/developer-role" target="_blank" rel="noopener noreferrer">go.calblueprint.org/developer-role</a> 👈</strong>
       designer_description: Learn what it means to be a designer <strong>👉 <a href="https://go.calblueprint.org/designer-role" target="_blank" rel="noopener noreferrer">go.calblueprint.org/designer-role</a> 👈</strong>
       external_description: Learn what it means to be on external <strong>👉 <a href="https://go.calblueprint.org/external-role" target="_blank" rel="noopener noreferrer">go.calblueprint.org/external-role</a> 👈</strong>
       application_timeline:
         infosession:
           title: Infosession Series
           description: >
-            We will host a general in-person infosession on Tuesday 8/29, and there will be multiple virtual infosessions on Saturday 8/26 and Sunday 8/27. These sessions will give you a chance to ask questions and talk to our members in a comfortable setting!
-            Take a look at our <a href="https://go.calblueprint.org/infosession">infosession schedule</a>.
+            We will host a general in-person infosession on Friday 1/20, and there will be multiple virtual infosessions on Saturday 1/21 and Sunday 1/22. These sessions will give you a chance to ask questions and talk to our members in a comfortable setting!
```

Comment:
"We will host a general in-person infosession on Tuesday 1/23"

change Friday 1/20 to Tuesday 1/23